src/main/java/com/andreamazzarella/contact_manager/Age.java

```diff
@@ -0,0 +1,36 @@
+package com.andreamazzarella.contact_manager;
+
+import java.util.regex.Pattern;
+
+public class Age {
+    private final String age;
+
+    public Age(String age) {
+        age = removeAllWhiteSpace(age);
+
+        if (isValidNumber(age)) {
```

Comment:
If you get a moment, Joshua Blochs - Effective Java has a load of great points about what to do/not do in Java. He mentions not having logic in constructors that could cause it to throw an error. 

I would not use try/catch for validation. I think your logic needs to be more explicit,

```
if (Age.canBeParsed("1")`
    Age a = new Age("1")
  else 
    //handle invalid age